### PR TITLE
New package: python-sh-1.12.9

### PR DESCRIPTION
**Pull request by lafrenierejm**

### CODE REVIEW

File: srcpkgs/python-sh/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'python-sh'
+pkgname=python-sh
+version=1.12.9
+revision=1
+noarch=yes
+wrksrc="sh-${version}"
+build_style=python-module
+pycompile_module="sh"
```
Comment (Vaelatern) [inline srcpkgs/python-sh/template]: `="sh.py"`

File: srcpkgs/python-sh/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+distfiles="${PYPI_SITE}/s/sh/sh-${version}.tar.gz"
+checksum=579aa19bae7fe86b607df1afaf4e8537c453d2ce3d84e1d3957e099359a51677
+
+post_install() {
+	vlicense LICENSE.txt
+}
+
+python3-sh_package() {
+	noarch=yes
+	pycompile_module="sh"
```
Comment (Vaelatern) [inline srcpkgs/python-sh/template]: And here

**Comment by asergi** [comment]
#5665

File: srcpkgs/python-sh/template [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'python-sh'
+pkgname=python-sh
+version=1.12.9
+revision=1
+noarch=yes
+wrksrc="sh-${version}"
+build_style=python-module
+pycompile_module="sh.py"
+hostmakedepends="python-setuptools python3-setuptools"
```
Comment (asergi) [inline srcpkgs/python-sh/template]: `depends="python"`

File: srcpkgs/python-sh/template [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+checksum=579aa19bae7fe86b607df1afaf4e8537c453d2ce3d84e1d3957e099359a51677
+
+post_install() {
+	vlicense LICENSE.txt
+}
+
+python3-sh_package() {
+	noarch=yes
+	pycompile_module="sh.py"
+	short_desc="${short_desc/Python2/Python3}"
```
Comment (asergi) [inline srcpkgs/python-sh/template]: `depends="python3"`

**Comment by lafrenierejm** [comment]
Closing in preference of #5696.

**Comment by Vaelatern** [comment]
@lafrenierejm sure you got that number right?

**Comment by lafrenierejm** [comment]
@Vaelatern I did not have the number right;  thanks for catching that.  Maybe I shouldn't be doing things that require thought while I have this sinus infection...

**Pull request closed without merging**